src/PanZoom.js

```diff
@@ -503,17 +533,35 @@ class PanZoom extends React.Component<Props> {
     return scaleMultiplier
   }
 
-  getPinchZoomLength = (finger1, finger2) => {
+  getPinchZoomLength = (finger1: Touch, finger2: Touch) => {
     return Math.sqrt(
       (finger1.clientX - finger2.clientX) * (finger1.clientX - finger2.clientX) +
       (finger1.clientY - finger2.clientY) * (finger1.clientY - finger2.clientY)
     )
   }
 
-  autoCenter = (zoomLevel = 1) => {
+  getContainer = (): HTMLDivElement => {
```

Comment:
@mnogueron I used `React.createRef`. Instead of doing null checks everywhere I tried to use these methods for accessing the DOM elements, where it is required. What is your opinion on this? These methods should actually never throw.

Reply:
As you said it should never throw. I think it's good to keep it with the throw as it is not creating any breaking change. If it happens that means there is a flaw somewhere else.

It is a bit overkilled but it's better to have more safety than the library breaking for no explainable reason.